### DISCUSSION TITLE
"Trapped in NZ:P"

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#537] [Map Releases]
I'm not really one to hype things up. Essentially, this is a challenge/box map with a goofy story taking place between Der Riese and Kino der Toten, where the cast is trapped inside of an NZ:P box map by a mysterious force. The story is very crude, so stay away if that's not your thing.

I've made this map to highlight some recent additions/changes to NZ:P that are beneficial for map makers.

Key features:
- Unrepairable barricades
- One-time use Perk-A-Cola Machines
- Forced-linear progression
- Buyable ending
- 2 wall weapons
- Double Tap version 1.0
- Active storytelling(???)

This map was not made to be very feasible to beat, there's a lot of intentional bullshit, so be sure to make me look like a dunce and share records somewhere. If you're reading this the day of posting, you'll have to manually update your game components to the newest versions, or wait for the nightly tomorrow.

I hope this map gets you thinking about new maps you can create for yourself. Feel free to report issues, but I am not changing the difficulty curve. This is a demonstration.

![fte-20231216165528-0](https://github.com/nzp-team/nzportable/assets/26030250/6ae2fb44-04eb-4f67-a098-1bea2c773554)

Thanks as always to my [Patrons](https://patreon.com/cypressimplex):
- Rinse a Gateau Mandate
- Benjamin Engelhardt
- DerpedCrusader
- mochi
- Ryan Baldwin
- LEXX
- MissLavender

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/13694079/cypress_challenge.zip)
## [Map Source](https://github.com/nzp-team/nzportable/files/13694041/cypress_challenge.zip) (for map makers)

**Comment by DerpedCrusader** [comment]
I was wondering what the perk icon textures were for

Definitely adding this to my planned nzp stream soonish, I'm waiting on a few community members to release maps they're working on

**Comment by ElFish09** [comment]
No idea what happened here or if its my fault, but it keeps saying the intro dialogue nonstop and its annoying

**Reply by ElFish09** [reply]
Ended up deleting the intro audio and now the rest goes on normal
